### PR TITLE
Np make vm size an input to multiome

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

_Give your PR a **concise** yet **descriptive** title._
_Please explain the changes you made here._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any issues fixed, addressed, or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
_You can delete these instructions once you have written your PR description._

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/multiome/Multiome.wdl [hunk 1]
```diff
@@ -35,6 +35,8 @@ workflow Multiome {
         Array[File] atac_r1_fastq
         Array[File] atac_r2_fastq
         Array[File] atac_r3_fastq
+        # VM size used for several ATAC tasks
+        String vm_size = "Standard_M128s"
```
Comment (kayleemathews) [inline pipelines/skylab/multiome/Multiome.wdl:39]: Let's make sure to add these new inputs to their respective Overview docs

File: pipelines/skylab/multiome/atac.wdl [hunk 2]
```diff
@@ -30,6 +30,7 @@ workflow ATAC {
     Int num_threads_bwa = 128
     Int mem_size_bwa = 512
     String cpu_platform_bwa = "Intel Ice Lake"
+    String vm_size
```
Comment (kayleemathews) [inline pipelines/skylab/multiome/atac.wdl:33]: We should add this input to the ATAC Overview as well

Comment (nikellepetrillo) [reply pipelines/skylab/multiome/atac.wdl:33]: added, ty!

**Comment by nikellepetrillo** [comment]
retest this please

**Pull request merged**